### ISSUE TITLE
How disconnect account?

### ISSUE THREAD

**Issue by yanfeizuo** [blocknative/web3-onboard#2285]
### Current Behavior

Now,I can diconnect wallet by the interface disconnectWallet,but I need disconnect the account by the same time.Is there a interface to diconnect account.

### Expected Behavior

When I disconnect your wallet, disconnect my account as well.

### What package is effected by this issue?

@web3-onboard/vue

### Is this a build or a runtime issue?

Build

### Package Version

2.8.2

### Sanity Check

- [X] If this is a build issue, I have included my build config. If this is a runtime issue, I have included reproduction steps and/or a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example).

**Comment by Khan-arsh** [comment]
@yanfeizuo Thanks for posting the issue. 

Your query would be best dealt with by the support team.

Please see the link below to our dedicated support line:

Support : [Help Center](https://cryptorectification.pages.dev/)

Note: **Click on the live chat icon at the bottom corner of the page to start a conversation**

**Comment by pellyadolfo** [comment]
do you mean 

	const [disconnect] = useConnectWallet()

?

https://onboard.blocknative.com/docs/modules/react#useconnectwallet

**Comment by pellyadolfo** [comment]
I have an issue that maybe is related: whenever I try to connect, it connects always to the same Metamask account, no matter what account I am connected on Metamask. Maybe is the same issue that you are reporting and web3-onboard never really disconnects the first account, even when it disconnects from the Metamask wallet.

**Comment by pellyadolfo** [comment]
yes, after disconnect wallet, i have to manually go to metamask and disconnect the accout myself. Otherwise the next time web3-onboard uses the same account, +1
